chip-gosu-functions.rb

```diff
@@ -0,0 +1,8 @@
+require 'gosu'
+require 'chipmunk'
+
+class Numeric
```

Comment:
I wouldn't recommend adding this method here. Maybe define a module or a singleton class Utils where you add this.